rust/sdk/src/client.rs

```diff
@@ -115,8 +115,11 @@ impl HttpClient {
         Ok(token)
     }
 
-    /// Makes request to Actyx apis. On http authorization error tries to
-    /// re-authenticate and retries the request ten times with increasing delay.
+    /// Perform a request (to Actyx APIs).
+    /// If an authorization error (code 401) is returned, it will try to re-authenticate.
+    /// If the service is unavailable (code 503), this method will retry to perform the
+    /// request up to 10 times with exponentially increasing delay - currently,
+    /// this behavior is only available if the `with-tokio` feature is enabled.
```

Comment:
This text is good, but on a private method people won’t see it. I’d probably add it to each end-user method that calls this function.

Reply:
I can add on them as well, this is a rust doc and not a regular one because we're the first clients after all